boards/px4/fmu-v2/default.cmake

```diff
@@ -118,3 +118,8 @@ px4_add_board(
 		#rover_steering_control # Rover example app
 		#segway
 	)
+
+# remove optional flight task features from fmu-v2 to save flash memory
+list(APPEND flight_tasks_to_remove
```

Comment:
@MaEtUgR What's going to happen if someone with a pixhawk tries to enter orbit mode?

I'd really like to consider not doing this kind of thing unless we want to get into feature defines (remove all aspects of orbit across the entire codebase) and adding APIs that expose what's actually available (mavlink).

Reply:
@dagar I'm fine with enabling it for all platforms I just felt it helps with the prevalent flash problem on v2. You are free to remove these lines at any time we have enough flash.

> What's going to happen...

It's neither unsafe nor perfectly nice. On one hand the drone continues what it did before and the pilot will get the command rejected error which makes it obvious that orbit is not working. On the other hand it will show orbit mode in the title bar until the mode is switched again. I think the desired behavior is getting a command not supported error and no other reaction. That's clearly one of the requirements for the flight task failsafe enhancements I'm looking into.